UPGRADING.md

```diff
@@ -2,6 +2,10 @@
 
 This document describes breaking changes and how to upgrade. For a complete list of changes including minor and patch releases, please refer to the [changelog](CHANGELOG.md).
 
+## v3.0.1
+
+If you're running node v10 or above you need to upgrade to `leveldown@2.0.1` or above. In addition, if you need prebuilt binaries you should upgrade to `leveldown@3.0.1` or above.
```

Comment:
I prefer "you'll need X" over "you need to X". And "or above" is implied, can be left out I think. But if you change it to "at least X to successfully compile" it becomes a subtle way to nudge people towards v3 :)

> If you're running node v10 you'll need at least `leveldown@2.0.1` to successfully compile from source. For prebuilt binaries please upgrade to `leveldown@3.0.1`.